explainer.md

```diff
@@ -623,13 +640,15 @@ enum XREnvironmentBlendMode {
 dictionary XRRenderStateInit {
   double depthNear;
   double depthFar;
+  double verticalFieldOfView;
   XRLayer? baseLayer;
   XRPresentationContext? outputContext
 };
 
 [SecureContext, Exposed=Window] interface XRRenderState {
   readonly attribute double depthNear;
   readonly attribute double depthFar;
+  readonly attribute double verticalFieldOfView;
```

Comment:
Should this be nullable and only valid for sessions without physically-driven values?

Reply:
Ewe, yeah. I see what you're getting at now. If it's just stated as "verticalFieldOfView" developers may look at it thinking it's reporting the physically based field of view in cases where there is one, which we don't want.

So the behavior that you're suggesting that we would report it as `null` any time a physical FoV is present, but the dictionary would not accept a nullable value? That feels a little quirky to me, but workable.

Reply:
I think the core point of confusion is having to know if this is a recommended value that can be overridden, or a physically-driven value that must be used as-is. Would it help to make this explicit in naming?

For example, for the nullable approach, using `suggestedVerticalFieldOfView` as the field name would make it clearer that the application doesn't have to use it as-is.

Alternatively, if the goal is to simplify applications by always supplying a field of view, there could be a separate `allowFieldOfViewModification` boolean that's true for devices where it's just a suggestion. That seems a bit clunky though.

Reply:
This is an important point.  Providing a default value without indicating that it's arbitrary is a bad idea.

Would it be reasonable to simply omit the value if it's not "real"?  developers can then do something like

`var fov = rendState.verticalFieldOfView || 90;`

Reply:
I think there's a minor theoretical advantage to having the device and/or UA provide a default, for example they could adjust the suggested value up or down based on the screen dimensions and typical usage, but I'm not aware of any UA doing this or planning to do so.

How about `var fov = rs.fixedVerticalFieldOfView || rs.suggestedVerticalFieldOfView`, where the UA would supply one or the other but never both? I think this would make the difference fairly clear to avoid accidental misuse, and the burden on apps that don't care is still fairly small since they can just use this conditional as-is.